doc/apps/eval.rst

```diff
@@ -0,0 +1,70 @@
+.. _eval_command:
+
+********************************************************************************
+eval
+********************************************************************************
+
+The ``eval`` command is used to compare the ``Classification`` dimensions of two
```

Comment:
Right now it is comparing only `Classification`, correct? We should update the note to that effect in case people want to compare other dimensions